charts/camunda-platform/test/tasklist/configmap_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 Camunda Services GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tasklist
+
+import (
+	"camunda-platform-helm/charts/camunda-platform/test/golden"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestGoldenConfigmapWithElasticsearchUrl(t *testing.T) {
+	t.Parallel()
+
+	chartPath, err := filepath.Abs("../../")
+	require.NoError(t, err)
+
+	suite.Run(t, &golden.TemplateGoldenTest{
+		ChartPath:      chartPath,
+		Release:        "camunda-platform-test",
+		Namespace:      "camunda-platform-" + strings.ToLower(random.UniqueId()),
+		GoldenFileName: "configmap-elastic-url",
+		Templates:      []string{"charts/tasklist/templates/configmap.yaml"},
+		SetValues:      map[string]string{"global.elasticsearch.url": "http://elasticmaster.org"},
```

Comment:
:wrench: I guess I would create for this test a property-based test, as we have for the deployment https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/test/tasklist/deployment_test.go since the golden files have been more intended for default values. You can added then more easily other tests, where you set other properties/variables. But it also of course depends on how you want to continue with this approach :)

Reply:
@Zelldon good point :+1:
Fixed :heavy_check_mark: 

Do you know if we have a library for Camunda config so we can easily unmarshal apps config like [camunda.tasklist](https://github.com/camunda/camunda-platform-helm/blob/ad23f83961e9aec911c02a3bf20671021d60af1c/charts/camunda-platform/charts/tasklist/templates/configmap.yaml#L10)?

Reply:
Hm not sure I think tasklist is a spring boot app, not sure whether this helps you here.

Reply:
then we probably will need to create that structure in the future.